opm/material/common/Valgrind.hpp

```diff
@@ -33,6 +33,12 @@
 #include <valgrind/memcheck.h>
 #endif
 
+#if HAVE_VALGRIND
+#define OPM_VALGRIND_OPTIM_UNUSED OPM_OPTIM_UNUSED
+#else
+#define OPM_VALGRIND_OPTIM_UNUSED
```

Comment:
*arg*: you're right. that issue seems to transform into a whack-a-mole. fix pushed, thanks.